app/controllers/application_controller.rb

```diff
@@ -1,2 +1,2 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ApiController
```

Comment:
You did it at reverse ;)
ApplicationController should be you're main and ApiContoller should inherit from ApplcationController